src/main/scala/nl.knaw.dans.dd.prepub/Configuration.scala

```diff
@@ -24,7 +24,7 @@ import org.apache.commons.configuration.PropertiesConfiguration
 
 case class Configuration(version: String,
                          serverPort: Int,
-                         pidGeneratorBaseUrl: String,
+                         pidGeneratorBaseUrl: URI,
```

Comment:
Use URI so that malformed URL will show up at start up.

Reply:
Just tried it in [one of my own](https://github.com/DANS-KNAW/easy-fedora-to-bag/blob/fe2ce6472836bab8663a208ee5267af683507def/src/main/scala/nl/knaw/dans/easy/fedoratobag/Configuration.scala#L61) applications, but it only threw an exception when the field was missing. I might have missed something.

Reply:
The idea is to fail early. So if you specify `pid-generator.base-url=something that is not a valid url` then the service will fail at startup instead of later on when the pid generator is actually called. But maybe I am missing your point @jo-pol ..?

Reply:
I had the same idea, tested it this morning manually on deasy with `http:/-/` and other garbage in `application.properties` but no exception was thrown. Maybe I missed what is not valid?